src/pages/home/library/EditSong.tsx

```diff
@@ -88,13 +88,15 @@ const EditSong = () => {
               color="white"
               variant="outlined"
               width="icon"
+              disabled={ !getIsSongDeletable(mintingStatus) }
```

Comment:
I see you added this, we might be okay to remove the `!(isMinted || isPending)` check right above this to stay consistent